streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -192,7 +182,7 @@ public void shouldProcessViaThroughTopic() {
     }
     
     @Test
-    public void testMerge() {
+    public void ShouldMergeStreams() {
```

Comment:
nit: camel case

Reply:
Please fix this, too.

Reply:
It is just a renaming to make its name consistent with others, I will rename it with non-capitalized name.